### ISSUE TITLE
Create FileRepo module

### ISSUE THREAD

**Issue by marcus-grant** [marcus-grant/scout#7]
The `FileRepo` class is needed before we can start working on the first CLI.

- [ ] `file` table
  - `id`
    - **Primary Key**
    - **Unique** **Integer**.
  - `dir_id`
    - **Foreign Key** to `dir.id`.
    - Parent directory containing file.
  - `name`
    - **TEXT**, **NULL**.
    - Name of the file.
  - `mtime`
    - **INTEGER**, **NULL**.
    - Last modified time according to file system's `mtime` stat.
  - `md5`
    - **BLOB**, **NULL**.
    - MD5 hash of the file.
    - Should be indexed
      - Need to find out if TEXT is a better hash data type.
      - Also need to figure out if a BLOB is even indexable.

**Comment by marcus-grant** [comment]
Duplicate of #11, merged in #13 with last associated commit bd53aec
